it/uk/gov/hmrc/incorporatedentityidentificationfrontend/controllers/BusinessVerificationControllerISpec.scala

```diff
@@ -107,6 +108,20 @@ class BusinessVerificationControllerISpec extends ComponentSpecHelper with AuthS
 
           lazy val result = get(s"$baseUrl/$testJourneyId/start-business-verification")
 
+          result.status mustBe SEE_OTHER
+          result.header(LOCATION) mustBe Some(routes.RegistrationController.register(testJourneyId).url)
+        }
+      }
+      "store a verification state of FAIL and redirect to the registration controller" when {
```

Comment:
It might be worth verifying that the verification state has been stored properly. I think we had an issue with that on another part of the code.

Reply:
Checking the correct verification status is done through the method:
stubStoreBusinessVerificationStatus
It basically makes sure the call to the backend had the correct businessVerificationStatus in it.
I've checked the backend and that just stores what it is sent.

Reply:
Sorry, I mean a verify on the wiremock call after you've completed the test. What you're doing currently is stubbing the response, but not checking that the call out has happened. If for some reason you weren't mapping on the response from the store call then we'd have no idea with just a stub.

As the store call isn't returning anything we need to continue, it's quite easy to fire and forget and not realise (this happened elsewhere in the code).

Reply:
Ah, understood.  Sorted now